PhysicsTools/PatAlgos/python/tools/cmsswVersionTools.py

```diff
@@ -203,40 +204,40 @@ def apply( self ):
 
         # Debugging output
         if debug:
-            print '%s DEBUG: Called with...'%( self._label )
+            print('%s DEBUG: Called with...'%( self._label ))
             for key in self._parameters.keys():
-               print '    %s:\t'%( key ),
-               print self._parameters[ key ].value,
+               print('    %s:\t'%( key ), end=' ')
+               print(self._parameters[ key ].value, end=' ')
```

Comment:
is this ```, end=' '``` needed here?

Reply:
answer to self: yes, it's needed.
```print ' ',``` doesn't end with newline and its replacement should have ```end=' '``` to replace the default ```\n```